setup.cfg

```
@@ -47,3 +47,4 @@ max-line-length = 100
 per-file-ignores =
     # Don't complain about unused imports in __init__.py
     */__init__.py: F401
+    */column_names.py: E501
```

I'd write out what we ignore here because it's not clear from the error code without looking it up. For example:

```suggestion
    # Ignore max line length as some column names and descriptions are very long
    */column_names.py: E501
```
But not major!